traffic_portal/app/src/common/modules/form/deliveryService/form.deliveryService.HTTP.tpl.html

```diff
@@ -403,13 +403,13 @@ <h3>Current Value</h3>
                     <div class="form-group" ng-class="{'has-error': hasError(cacheConfig.maxRequestHeaderBytes), 'has-feedback': hasError(cacheConfig.maxRequestHeaderBytes)}">
                         <label class="has-tooltip control-label col-md-2 col-sm-2 col-xs-12" for="maxRequestHeaderBytes">Max Request Header Bytes<div class="helptooltip">
                                 <div class="helptext">
-                                    The maximum size (in bytes) of the request headers for this particular delivery service. Clearing this will result in the default (131072) being set for this DS.
+                                    The maximum size (in bytes) of the request headers for this particular delivery service. Clearing this will result in the default (0) being set for this DS.
```

Comment:
You might want to tell the user what a value of 0 means exactly. Does it mean no max?

Reply:
Good call, we might even want to reference the ATS parameter that this corresponds to: https://docs.trafficserver.apache.org/en/8.0.x/admin-guide/files/records.config.en.html#proxy-config-http-request-header-max-size.